web-api/migration-terraform/main/lambdas/migrations/0023-set-served-docket-entries-as-completed.js

```diff
@@ -0,0 +1,31 @@
+const createApplicationContext = require('../../../../src/applicationContext');
+const {
+  DocketEntry,
+} = require('../../../../../shared/src/business/entities/DocketEntry');
+const {
+  DOCUMENT_PROCESSING_STATUS_OPTIONS,
+} = require('../../../../../shared/src/business/entities/EntityConstants');
+const applicationContext = createApplicationContext({});
+
+const migrateItems = async items => {
+  const itemsAfter = [];
+  for (const item of items) {
+    if (
+      item.pk.startsWith('case|') &&
+      item.sk.startsWith('docket-entry|') &&
+      item.servedAt &&
+      item.processingStatus !== DOCUMENT_PROCESSING_STATUS_OPTIONS.COMPLETE
+    ) {
+      item.processingStatus = DOCUMENT_PROCESSING_STATUS_OPTIONS.COMPLETE;
+
+      new DocketEntry(item, { applicationContext }).validate();
+
+      itemsAfter.push(item);
```

Comment:
can we please log the pk/sk to the .logger for any operation where we make this change